draft-ietf-quic-transport.md

```diff
@@ -3141,7 +3179,8 @@ is increased.
 The final offset is the count of the number of octets that are transmitted on a
 stream.  For a stream that is reset, the final offset is carried explicitly in
 the RST_STREAM frame.  Otherwise, the final offset is the offset of the end of
-the data carried in STREAM frame marked with a FIN flag.
+the data carried in STREAM frame marked with a FIN flag, or 0 in the case of
+incoming unidirectional streams.
 
```

Comment:
in _the_ STREAM?

Reply:
Detail, but the RST_STREAM just above is referenced as _the_ RST_STREAM - is that intentional?